sklearn/metrics/classification.py

```diff
@@ -892,6 +903,12 @@ def f1_score(y_true, y_pred, labels=None, pos_label=1, average='binary',
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    zero_division : string or int, default="warn"
+        Sets the behavior when there is a zero division. If set to
+        ("warn"|0)/1, returns 0/1 when both precision and recall are zero
```

Comment:
I don't think this notation is easy enough to read. How about 'Sets the value to return when blah blah. If "warn" (default), this acts like `0` but also raises a warning.'

Reply:
wrote something similar, please check new version